templates/nsg.tftpl

```diff
@@ -0,0 +1,7 @@
+# NSG
+module "nsg" {
+  source         = "qbeyond/nsg/azurerm"
+  resource_group = azurerm_resource_group.network
+  version        = "0.0.1"
+  yaml_path      = "./nsg.yaml"
+}
```

Comment:
As mentioned above I think we should no longer use the YAML module

Reply:
Yes, I agree with you. However, until we don't have other module, or new module version, we prefer continue to use.

Reply:
We decided to just not do a module for it since there are barely any shared code components between these NSG modules. It should just be plain terraform code in my opinion.